### ISSUE TITLE
Physical air handling unit model

### ISSUE THREAD

**Issue by MBaranskiEBC** [RWTH-EBC/AixLib#357]
alexanderAKU and myself agreed on creating a detailed physical model of an air handling unit because the existing AHU model (though it's a great model) 
- is based on simplifying assumptions
- combines physical world and control
- cannot be easily extended by modules such as sorption

As a first step, I will include a model we already have in a local repo. As a next step, we could create a new project and develop a package for physical AHU models.

**Comment by marcusfuchs** [comment]
@MBaranskiEBC @alexanderAKU
Please check with @KFinkbeiner how this is related to #310.

**Comment by KFinkbeiner** [comment]
Please check, if the modular modelling approach and the description of related physics by characteristical maps of moist air is suitable for you. The integration of generic controller into each module can be a helpful feature for the users of this model. We need to check, if a collaboration is possible.

**Comment by marcusfuchs** [comment]
@KFinkbeiner 
Are you (and your student) already actively working on this? If so, could you push your current work to a branch (e.g. named `issue310_modularAHU`) so that @MBaranskiEBC and @alexanderAKU can check how these efforts may be combined?

**Comment by alexanderAKU** [comment]
@MBaranskiEBC @KFinkbeiner 
We should discuss our work and aims in a meeting

**Comment by KFinkbeiner** [comment]
Currently we identified the individual components of an air handling unit and their diverse technologies, which are used for heating, cooling, humidification and dehumidification as well as heat and moist recovery. For these components and their related technologies, we described the characteristical behaviour in the Mollier diagram. The next step is the modelling of modules. At the moment, my student is getting familiar with Dymola/Modelica.
